### PR TITLE
TSK-1843: refactor GetTaskCommentAccTest to use test-api

### PR DESCRIPTION
**Pull request by ryzheboka**
https://sonarcloud.io/summary/new_code?id=ryzheboka_taskana&branch=TSK-1843

---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+  void should_ReturnEmptyList_When_TaskCommentsDontExist() throws Exception {
+    assertThat(taskService.getTaskComments(task2.getId())).isEmpty();
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_FailToReturnTaskComments_When_TaskIstNotVisible() {
+    assertThatThrownBy(() -> taskService.getTaskComments(task1.getId()))
+        .isInstanceOf(NotAuthorizedException.class);
+  }
```
Comment (ensarevlideveloper) [inline lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:128]: I think we want to check Exception messages. How about building a NotAuthorizedException expectedException and then asserting equality? :)

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    assertThat(taskComment).isEqualTo(comment);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_FailToReturnTaskComment_When_TaskCommentIsNotExisting() {
+    assertThatThrownBy(() -> taskService.getTaskComment("Definately Non Existing Task Comment Id"))
+        .isInstanceOf(TaskCommentNotFoundException.class);
+  }
+
```
Comment (ensarevlideveloper) [inline lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:152]: See above :)

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_FailToReturntaskComment_When_TaskIstNotVisible() throws Exception {
+    TaskComment comment =
```
Comment (ensarevlideveloper) [inline lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:156]: Small naming convention mistake in word task.

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+        TaskCommentBuilder.newTaskComment()
+            .taskId(task1.getId())
+            .textField("Text1")
+            .created(Instant.now())
+            .modified(Instant.now())
+            .buildAndStore(taskService, "user-1-1");
+
+    assertThatThrownBy(() -> taskService.getTaskComment(comment.getId()))
+        .isInstanceOf(NotAuthorizedException.class);
+  }
```
Comment (ensarevlideveloper) [inline lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:166]: See above.

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java (hunk 1 above)
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:128]: Ok, I will check the text of the error message

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java (hunk 2 above)
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:152]: Ok

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java (hunk 3 above)
Comment (ryzheboka) [reply lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:156]: Ups :)

**Review by ensarevlideveloper — APPROVED** [review]

File: lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+    TaskComment taskComment = taskService.getTaskComment(comment.getId());
+
+    assertThat(taskComment).isEqualTo(comment);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_FailToReturnTaskComment_When_TaskCommentIsNotExisting() {
+    assertThatThrownBy(() -> taskService.getTaskComment("Definately Non Existing Task Comment Id"))
+        .isInstanceOf(TaskCommentNotFoundException.class);
```
Comment (ensarevlideveloper) [inline lib/taskana-core-test/src/test/java/acceptance/taskcomment/get/GetTaskCommentAccTest.java:179]: Please use the assertions methods as used before :)

**Review by ensarevlideveloper — CHANGES_REQUESTED** [review]

**Review by ensarevlideveloper — APPROVED** [review]

**Pull request merged**